include/logger.hpp

```diff
@@ -118,7 +118,7 @@ void LOG_set_logs_allowed_function(LogsAllowedCallback logs_allowed_function);
 #define LG_IF_LVL_OK(level, ...)                                               \
   do {                                                                         \
     if (unlikely(LOG_is_logging_enabled_for_level(level))) {                   \
-      olprintfln(ABS(level), -1, MYNAME, __VA_ARGS__);                         \
+      olprintfln(ABS(level), -1, nullptr, __VA_ARGS__);                        \
```

Comment:
Minor:
It is slightly strange to put a nullptr here.
I understand that we default to the context name